static/templates/subscription_settings.handlebars

```diff
@@ -18,6 +18,8 @@
     </div>
     <button class="button small white rounded subscribe-button sub_unsub_button {{#unless subscribed }}unsubscribed{{/unless}}" type="button" name="button">
         {{#if subscribed }}Unsubscribe{{else}}Subscribe{{/if}}</button>
+    <a href="{{preview_url}}"><button class="button small white rounded preview-button" type="button" name="button">
```

Comment:
This doesn't seems to be a valid HTML. See here: http://stackoverflow.com/q/6393827. You may want to do something like this:`<a href="{{preview_url}}" class="button small white rounded preview-button">{{t "View stream"}}</a>`.
You will need to make adjustments to css.